### ISSUE TITLE
RAWG 504 Error

### ISSUE THREAD

**Issue by m1cypher** [CMorooney/obsidian-game-search-plugin#28]
So when adding a game today, it would just error out with a 504 error. My fix was the log into RAWG and then the create game note worked. Might be worth putting into the README that troubleshooting step.

**Comment by CMorooney** [comment]
Thanks for reporting an issue!

I'm a little confused though -- You're saying that after logging into the RAWG website your obsidian plugin stopped giving you 504 errors 🤔?

504 is a timeout error. [I don't see any reported outages at RAWG recently](https://status.rawg.io/) is it possible your internet connection was momentarily out?

**Comment by m1cypher** [comment]
You are absolutely correct. I checked the same site and was able to reach it and saw that nothing was down. I then attempted to login, it took a couple times which I have learned isn't uncommon, and after I logged in I went back to obsidian and the creation worked. Below is the steps that I took so it is more clear.

1) Opened command pallet and started game search for new note
2) Searched the game I wanted
3) List of games from search appeared and I click on the one I wanted
4) No new note was created
5) Checked the console and say a 500 error for the plugin. (I believe it was 504, but I can't remember at this time)
6) Since 500 is a server side error, I went to rawg.io and saw that the site was up.
7) Googled rawg status and found the status page with green across the board
8) Figured maybe there was some terms and conditions that might need to be agreed to so I logged in
9) Started a new search in obsidian and search appeared, new note was created

Does this all make sense?
